artiq/tools.py

```diff
@@ -114,6 +114,23 @@ def verbosity_args(parser):
 
 
 def simple_network_args(parser, default_port):
+    """    Add simple network arguments to a command-line argument parser.
```

Comment:
Why the spaces at the beginning?

Reply:
Fixed. Formatting mistake when removing ``\n`` after ``"""``